### PR TITLE
Adding metadata to line_items

### PR DESCRIPTION
**Pull request by aleksen**

### CODE REVIEW

File: mcash/mapi_client/validation.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ def wrapper(*args, **kwargs):
     'line_items': Any([{
         Required('product_id'): basestring,
         'vat': basestring,
+        'metadata': Any([{'key': basestring, 'value':basestring}],None),
```
Comment (mandus) [inline mcash/mapi_client/validation.py]: missing a space in 'value': basestring

File: mcash/mapi_client/validation.py [hunk 2]
```diff
@@ -106,6 +107,7 @@ def wrapper(*args, **kwargs):
...
```
Comment (mandus) [inline mcash/mapi_client/validation.py]: same..

**Comment by mandus** [comment]
LGTM (except minor detail)

**Comment by aleksen** [comment]
@mandus All pepped up!

**Pull request merged**